### PR TITLE
Pass triggered_after_koji_task parameter through to osbs-client

### PR DESCRIPTION
**Pull request by rcerven**
* OSBS-8149

Signed-off-by: Robert Cerven <rcerven@redhat.com>

### CODE REVIEW

**Comment by MartinBasti** [comment]
Tests fix: https://github.com/containerbuildsystem/koji-containerbuild/pull/137

File: koji_containerbuild/plugins/builder_containerbuild.py [hunk 1]
```diff
@@ -338,6 +338,15 @@ class BuildContainerTask(BaseTaskHandler):
...
                         "type": ["string", "null"],
                         "description": "Signing intent of the ODCS composes."
+                    },
+                    "skip_build": {
+                        "type": "boolean",
+                        "description": "Skip build, just update buildconfig for autorebuild "
+                                       "and don't start build"
+                    },
+                    "triggered_after_koji_task": {
+                        "type": ["integer", "null"],
```
Comment (twaugh) [inline koji_containerbuild/plugins/builder_containerbuild.py]: What would `"triggered_after_koji_task": null` mean?

Comment (rcerven) [reply koji_containerbuild/plugins/builder_containerbuild.py]: basically the same as it wasn't defined, I mean I guess I can have there just 'integer' ?

Comment (twaugh) [reply koji_containerbuild/plugins/builder_containerbuild.py]: I think I'd prefer just 'integer' for minimum ambiguity.

Comment (rcerven) [reply koji_containerbuild/plugins/builder_containerbuild.py]: changed

**Pull request merged**